ui_framework/components/local_nav/_index.scss

```diff
@@ -17,6 +17,12 @@ $localTabTextColor-isSelected: $localNavButtonTextColor-isHover;
 $localBreadcrumbLinkColor: #328caa;
 $localSearchButtonBackgroundColor: #9c9c9c;
 $localSearchButtonTextColor: #ffffff;
+$localCalendarButtonHoverBackgroundColor: $localNavButtonBackgroundColor-isHover;
+$localCalendarCellPrecludedBackgroundColor: rgba(black, 0.05);
+$localCalendarCellSelectedBackgroundColor: #777777;
+$localCalendarCellSelectedTextColor: #ffffff;
+$localCalendarCellHoverBackgroundColor: $globalInfoColor;
```

Comment:
If that's the case, then use a cursor:not-allowed. I assumed it was because of the pointer.